### DISCUSSION TITLE
New iteration of MOD metadata review should cover sdo:DefinedTermSet

### DISCUSSION THREAD

**Discussion by jonquet** [sifrproject/MOD-Ontology#51] [Ideas]
Since MOD 1.4 Schema.org (SDO) has added the notion of DefinedTerm and [DefinedTermSet](https://schema.org/DefinedTermSet) (another word for semantic artefact I believe). 

We are going to need to review the metadata properties of this object. 
Things inherited from CreativeWork and Thing were already integrated in MOD1.4 which assume an ontology was a sdo:Dataset

Note with respect to #34 => we might have a conflict when aligning our DCAT MOD2 approach with Schema.org. 
@agbeltran : DCAT3 is aligned to sdo:DataSet and if we decide a mod:SemanticAretfact is a dcat:Dataset we are going to have an issue of alignement with Schema.org in which the relevant class is now sdo:DefinedTermSet which is disjoint from sdo:Dataset

**Comment by agbeltran** [comment]
hi @jonquet - thansk for the heads up on this. This is a good point. I think we are going with the option of deriving from dcat:Resource (even though we will still use the distribution property, so by reasoning, mod:SemanticArtefact will be a dcat:Dataset) so this should be OK
